src/test/resources/journeys.conf

```diff
@@ -1,4 +1,16 @@
-# Configure here your journeys. A journey is a sequence of requests at a certain load.
```

Comment:
Could we  keep this comment, after the HMRC license info?

Reply:
Thank you for pointing that. I have fixed that now.